mysite/blog/tests/test_views.py

```diff
@@ -0,0 +1,9 @@
+from blog.tests.test_modelmixintestcase import ModelMixinTestCase
+from django.test import TestCase
+from django.urls import reverse
+
+
+class TestListView(ModelMixinTestCase, TestCase):
+    def test_post_list_GET(self):
+        template = self.client.get(reverse("blog:post_list"))

```

Comment:
This is response not template

```suggestion
        response = self.client.get(reverse("blog:post_list"))

```